### ISSUE TITLE
Speak enumeration signs and bullet shapes in MS Word

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#7742]
[production process.docx](https://github.com/nvaccess/nvda/files/1464347/production.process.docx)
[symbole.zip](https://github.com/nvaccess/nvda/files/1464351/symbole.zip)
Steps to reproduce:
1. Open MS word and type following:
1. production process
1.1. planning
* Example
- part one
- part two
2. starting process
2.1. equipment requirements
a. raw materials
b. machines
c. packages

Expected behavior:
NVDA should read on each line the level of enumeration and the shape of bullets. More exactly, MS word converts each time the numbers and bullets in text font symbols or windings. Those text fonts are not described in NVDA. There is a word document in german which provided descriptions only for windings in MS word before 2003 but it is certainly not up to date anymore. I have inserted this document just for your information. 

Actual behavior:
If you navigate by arrow keys, NVDA speaks only the text after the bullet / numbers. If you place the cursor on the first letter on a line and press the left arrow key to move to the bullet or number, NVDA freezes for some mili seconds and just repeats the first letter instead of reading the type of bullet or number. The issue occurs indifferent which language / speech API / level of punctuation is being selected.

System configuration:
NVDA version:
2017.3

NVDA Installed or portable:
both

Other information:
Windows version:
all versions

Name and version of other software in use when reproducing the issue:
Office all Versions since 2007

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
Every Version since 2013.2, same issues.

**Comment by Adriani90** [comment]
This issue occurs in all office programs (ms excel, power point etc.)

**Comment by feerrenrut** [comment]
It's not obvious to me how this differs from #7804, as I commented on that issue, could you please update the expected and actual steps to include the speech you get and expect from NVDA this helps to complete the example.

**Comment by Adriani90** [comment]
I have edited the steps to reproduce and the behaviour section. Hope you can see the difference now.

**Comment by feerrenrut** [comment]
Are you suggesting that for MS Word NVDA should not honour the users "lists" setting, and always report the lists? Can you give some justification for why this should be the case?

**Comment by Adriani90** [comment]
Not only in ms word, but in outlook and power point as well. Because people often  don‘t need the information „list with x entries“ „list end“ which is always reported in browsers when lists are enabled in document formating.
And I would even expand this issue to libre office and otheer text editors which automatically generate lists when inserting bullets and enumerations.


Von meinem iPhone gesendet

**Comment by feerrenrut** [comment]
I'm sorry, I'm not following the request.

Are you suggesting that "list with 5 entries" and "list end" should honour the lists setting in document formatting dialog but "bullet", "dash", "one", and "one point one"  should not honour this setting and always be reported?

**Comment by Adriani90** [comment]
Yes, that‘s right.


Von meinem iPhone gesendet

**Comment by feerrenrut** [comment]
This would suggest that its invalid for a user to wish not to have "bullet etc" announced.

**Comment by Adriani90** [comment]
Because it is much more important to get the structure within the list rather than where the list starts and where it ends.


Von meinem iPhone gesendet

**Comment by leonardder** [comment]
I agree with this request. I think that the right behavior here would be that NVDA reports the bullet based on the symbol level, regardless of whether reporting of lists is on or off. This is how it works on the web as well.

**Comment by feerrenrut** [comment]
> This is how it works on the web as well.

@leonardder thanks for pointing that out.

**Comment by Adriani90** [comment]
A user could disable an additional setting called bullets and enumerations in document formating settings. But I guess not many would disable it since it can be very important for keeping the orientation during reading.


Von meinem iPhone gesendet

**Comment by feerrenrut** [comment]
I would prefer not to introduce another setting for this if we can avoid it. I have invited community members to comment on this issue. If there is no serious objection to the request then it we can pursue it without making it optional.

**Comment by leonardder** [comment]
@feerenrut, I think that, from an implementation perspective, this is very similar to the office side of #4837.
To summarize, disabling reporting of graphics is not yet supported by NVDA as this feature is blocked by how office deals with graphics. When you disable reporting of graphics, also the alternative texts of graphics disappear altogether.

**Comment by derekriemer** [comment]
I agree

**Comment by feerrenrut** [comment]
Thanks @leonardder, that issue is quite relevant, and may have an impact on how this one is implemented. The way I read #4837, it would be possible to stop the announcement of "graphic" but will require some careful implementation, and possibly some changes in overall design.

**Comment by feerrenrut** [comment]
@dkager Yep, happy for that to happen.
